src/main/java/Duke.java

```diff
@@ -1,10 +1,166 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.time.LocalDate;
+
 public class Duke {
+
+    static ArrayList<Task> list = new ArrayList<Task>();
+
+    public static void handle(String input) throws DukeException{
+        String[] split = input.split(" ");
+        String command = split[0];
+        if (command.equals("todo")) {
+            if (split.length == 1) {
+                throw new DukeException("It looks like you're missing the task description");
+            } else {
+                Todo t = new Todo(input.substring(5));
+                list.add(t);
+                System.out.println("Got it. I've added: \n" + t.toString());
+                System.out.printf("Sheesh you've now got %d tasks in the list\n", list.size());
+            }
+        } else if (command.equals("deadline")) {
+            if (split.length == 1) {
+                throw new DukeException("It looks like you're missing the task description");
+            } else {
+                String[] arr = input.substring(9).split("/by ");
+                if (arr.length == 1) {
+                    throw new DukeException("It seems you haven't included the deadline");
+                } else {
+                    String[] date = arr[1].split("-");
+                    if (date.length != 3) {
+                        throw new DukeException("Your deadline should be in the format yyyy-mm-dd");
+                    }
+                    Deadline d = new Deadline(arr[0],
+                            LocalDate.of(Integer.parseInt(date[0]),
+                                    Integer.parseInt(date[1]),
+                                    Integer.parseInt(date[2])));
+                    list.add(d);
+                    System.out.println("Got it. I've added: \n" + d.toString());
+                    System.out.printf("Sheesh you've now got %d tasks in the list\n", list.size());
+                }
+            }
+        } else if (command.equals("event")) {
+            if (split.length == 1) {
+                throw new DukeException("It looks like you're missing the task description");
+            } else {
+                String[] arr = input.substring(6).split("/at ");
```

Comment:
Would adding a `name/variable` to `input.substring(6)` be better for code readability?